OpenRA.Mods.Common/Widgets/Logic/ServerListLogic.cs

```diff
@@ -218,6 +219,14 @@ public ServerListLogic(Widget widget, ModData modData, Action<GameServer> onJoin
 				}
 			}
 
+			var playersLabel = widget.GetOrNull<LabelWidget>("PLAYER_COUNT");
+			if (playersLabel != null)
+			{
+				var playersText = new CachedTransform<int, string>(c => c.ToString() + " Players Online");
```

Comment:
Nit: This will yield `1 Players Online` if `c` is 1.

Please change this to either:
- ` Player(s) Online`, or
- ternary `Player` vs `Players`

Reply:
Good point. Fixed.